docs/supported_apis/index.rst

```diff
@@ -1,49 +1,30 @@
-Supported APIs and Defaulting to pandas
-=======================================
+Supported APIs
+==============
 
 For your convenience, we have compiled a list of currently implemented APIs and methods
 available in Modin. This documentation is updated as new methods and APIs are merged
 into the master branch, and not necessarily correct as of the most recent release. In
```

Comment:
we can point here to a `stable` version of ReadTheDocs for Modin, which refers to the latest release.

Reply:
Done.